tests/modules/nf-scil/registration/tractogram/nextflow.config

```diff
@@ -0,0 +1,15 @@
+process {
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
```

Comment:
remove this line, else you include files from LOAD_TEST_DATA in your test outputs